Objects/odictobject.c

```diff
@@ -887,14 +887,20 @@ PyDoc_STRVAR(odict_init__doc__,
 \n\
         ");
 
+static int odict_tp_init(PyObject *, PyObject *, PyObject *);
 /* forward */
-static int odict_init(PyObject *self, PyObject *args, PyObject *kwds);
+static PyObject *
+odict_init(PyObject *self, PyObject *Py_UNUSED(ignored))
```

Comment:
I suggest to left this for a separate issue.

Reply:
OK, I'll drop this and the other patch from this PR and make a different one.